features/guidance/dedicated-turn-roads.feature

```diff
@@ -71,19 +71,19 @@ Feature: Slipways and Dedicated Turn Lanes
     Scenario: Turn Instead of Ramp
         Given the node map
             """
-                    e
-            a b     c d
-                  h
-
-
-
-
-
-
-
+                    e---.
+            a-b-----c-d |
+               `--h     |
+                   \    /
+                    |  /
+                    | /
+                    |/
+                    |
+                    |
+                    |
                     f
```

Comment:
while technically correct (e does not connect to `c` due to a bridge, the road looks strange with this angles.

how about adding `] c [` style chars here? to indicate elevation?